### ISSUE TITLE
Undo and Redo

### ISSUE THREAD

**Issue by BlessedGlen** [Slipity/App#16]
Hello, will it be possible to have undo and redo option, thank u

**Comment by josiaho** [comment]
Good idea! Where would see this best functioning? While "editing", or in some global capacity?

**Comment by BlessedGlen** [comment]
While editing and re arranging the slides if it’s possible, thank u so much.
